features/step_definitions/antivirus_steps.rb

```diff
@@ -0,0 +1,28 @@
+require 'aws-sdk-s3'
+
+Given /^I have a randomized file continaing the eicar test signature$/ do
+  @file = Fixtures.eicar_test_signature + SecureRandom.base64(129)
+end
+
+When /^I upload #{MAYBE_VAR} to the documents bucket under the key #{MAYBE_VAR}$/ do |file_contents, destination_key|
+  s3 = Aws::S3::Resource.new(region: 'eu-west-1')  # actual region is unimportant
+  @s3_obj = s3.bucket(dm_documents_bucket_name).object(destination_key)
+  @s3_obj_response = @s3_obj.put(
```

Comment:
Does Jenkins definitely have `put` permissions for the bucket, and do we need any permissions to be able to retrieve it later - I only ask because of my trauma in this area this past week 😸

Reply:
This is a good question to which I don't know the answer. @Wynndow ?